ujes/definedEngines/flink/engine/pom.xml

```diff
@@ -0,0 +1,309 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ~ Copyright 2019 WeBank ~ ~ Licensed under the Apache License, Version 
+    2.0 (the "License"); ~ you may not use this file except in compliance with 
+    the License. ~ You may obtain a copy of the License at ~ ~ http://www.apache.org/licenses/LICENSE-2.0 
+    ~ ~ Unless required by applicable law or agreed to in writing, software ~ 
+    distributed under the License is distributed on an "AS IS" BASIS, ~ WITHOUT 
+    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. ~ See the 
+    License for the specific language governing permissions and ~ limitations 
+    under the License. -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>linkis</artifactId>
+        <groupId>com.webank.wedatasphere.linkis</groupId>
+        <version>0.11.0</version>
+        <relativePath>../../../../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>linkis-flink-engine</artifactId>
+    <properties>
+        <flink.version>1.11.1</flink.version>
+        <hadoop.version>2.6.0-cdh5.9.2</hadoop.version>
+        <scala.macros.version>2.0.1</scala.macros.version>
+        <scala.binary.version>2.11</scala.binary.version>
+        <scala.version>2.11.12</scala.version>
```

Comment:
Better to quote scala version in linkis parent pom. Or you may get two scala jar with different versions in final package, which may cause unexpected problems.